src/Microsoft.AspNetCore.SignalR.Redis/Internal/RedisChannels.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.AspNetCore.SignalR.Redis.Internal
+{
+    internal class RedisChannels
+    {
+        private readonly string _prefix;
+
+        /// <summary>
+        /// Gets the name of the channel for sending to all connections.
+        /// </summary>
+        /// <remarks>
+        /// The payload on this channel is <see cref="RedisInvocation"/> objects containing
+        /// invocations to be sent to all connections
+        /// </remarks>
+        public string All { get; }
+
+        /// <summary>
+        /// Gets the name of the internal channel for group management messages.
+        /// </summary>
+        public string GroupManagement { get; }
+
+        public RedisChannels(string prefix)
+        {
+            _prefix = prefix;
+
+            All = $"{prefix}:all";
+            GroupManagement = $"{prefix}:internal:groups";
+        }
+
+        /// <summary>
+        /// Gets the name of the channel for sending a message to a specific connection.
+        /// </summary>
+        /// <param name="connectionId">The ID of the connection to get the channel for.</param>
+        public string Connection(string connectionId)
+        {
+            return $"{_prefix}:connection:{connectionId};";
```

Comment:
Are these used a lot? Normal string concatenation would be faster.

Reply:
This would be called once any time a user sends to a specific connection. I can switch to normal string concat though.

Reply:
Yes, lets concat